student.rb

```diff
@@ -1,11 +1,11 @@
 require_relative 'person'
 
 class Student < Person
-  attr_reader :classroom
+  attr_accessor :classroom
 
-  def initialize(classroom:, name: 'Unknown', age: nil, parent_permission: true)
+  def initialize(name: 'Unknown', age: nil, parent_permission: true)
     super(name: name, age: age, parent_permission: parent_permission)
-    @classroom = classroom
+    @classroom = nil
   end
```

Comment:
- [ ] Good job with associations 👏 , as for the Student class that take in charge student , students should belong to a classroom, Kindly add a method that serves as the 'belongs-to' side (a student belongs to a classroom).


```ruby
class Student < Person
  attr_reader :classroom
  .
  .
  .
  def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
end
```

For more information about this step , kindly follow this[ article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) 👍